deepsea.spec.in

```diff
@@ -22,7 +22,7 @@
 # See also http://en.opensuse.org/openSUSE:Shared_library_packaging_policy
 
 Name:           deepsea
-Version:        0.9
+Version:        DEVVERSION
```

Comment:
`@DEVVERSION@` would make it more obvious that this is a token designated for replacement

Reply:
While it doesn't bother me, the multiple meanings of '@' in a Makefile might make others think that the double @ denotes some special processing.  I'd say open another issue for the variable naming in the spec and Makefile.